src/databricks/labs/ucx/source_code/python_libraries.py

```diff
@@ -5,6 +5,12 @@
 from collections.abc import Callable
 from functools import cached_property
 from pathlib import Path
+from zipimport import ZipImportError
```

Comment:
Pylint says this error does not exists, but it does, dunno what is happening here